utilities.py

```diff
@@ -48,6 +65,13 @@ def clean_tmp_data_from_device(device, remove_page_data=True, remove_pids_data=T
 
 
 def clean_tmp_data(remove_page_data=True, remove_pictures_data=True, remove_pids_data=True):
+    """Removes collected from a device and transferred into app pagedata
```

Comment:
"apagedata" instead of "pagedata"

Reply:
What do you mean by `apagedata`? Here `pagedata` means internal data, stored in application